src/Util/MissingFontTexture.cpp

```diff
@@ -0,0 +1,18 @@
+#include <SDL.h>
+#include <SDL_image.h>
+#include <Util/Logger.hpp>
+
+#include "Util/MissingFontTexture.hpp"
+
+SDL_Surface *GetMissingFontTextureSDLSurface() {
```

Comment:
This function is 99%-similar to `GetMissingImageTextureSDLSurface()`; could they merge into one function? Also, they should be in namespace `Util` IMO.

Reply:
The reason why I dont merge two function into one function:
 - It's hard to make two functions into one function.
 - It's hard to use the merged function and specific it is font texture or image texture.

Based on two reason, I think it should not merged into one function.

Reply:
The namspace `Util` should be fixed soon.